gatsby-config.js

```diff
@@ -20,10 +21,11 @@ const languages = [
 
 module.exports = {
   siteMetadata: {
-    title: `Chicago COVID Resources`,
+    siteTerritory: process.env.SITE_TERRITORY,
```

Comment:
this was the best name I could up with, open to other options. At first I was working with CITY, but its possible someone might want to use this for a larger/smaller area than that!